### PR TITLE
EPRS: added PG and EPAS 17 support

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?
Short PR that adds PostgreSQL 17 and EPAS 17 to the supported database versions. 

https://enterprisedb.atlassian.net/browse/DOCS-975 

This PR can be merged to the eprs_7.10 release branch, where the content will be staged until release.

### CODE REVIEW

File: product_docs/docs/eprs/7/01_introduction/03_certified_supported_versions.mdx [hunk 1]
```diff
@@ -16,7 +16,7 @@ You can use the following database product versions with Replication Server:
 
 All PostgreSQL and EDB Postgres Advanced Server versions available as EDB Postgres AI Cloud Service single-node and primary/standby high-availability cluster types are also supported for SMR configurations. See the Cloud Service [documentation](/edb-postgres-ai/cloud-service/) for more information about Cloud Service’s [supported cluster types](/edb-postgres-ai/cloud-service/references/supported_cluster_types/). See the [database version policy documentation](/edb-postgres-ai/cloud-service/references/supported_database_versions/) for the versions of PostgreSQL and EDB Postgres Advanced Server available in Cloud Service.
 
-EDB Postgres Distributed (PGD) v5.3.0 is explicitly certified as a Publishing database for trigger mode and as Subscription database for both trigger and wal modes.
+EDB Postgres Distributed (PGD) v5.6.0 is explicitly certified as a Publishing database for trigger mode and as Subscription database for both trigger and wal modes.
```
Comment (gvasquezvargas) [inline product_docs/docs/eprs/7/01_introduction/03_certified_supported_versions.mdx:19]: @mlewandowski-edb @zahid75 
For PGD, we currently are at 5.6. 
Have we certified EPRS with 5.6 or any version later than 5.3? 

If not, I can roll back this change.

File: product_docs/docs/eprs/7/01_introduction/03_certified_supported_versions.mdx [hunk 2]
```diff
@@ -4,8 +4,8 @@ title: "Certified and supported product versions"
 
 You can use the following database product versions with Replication Server:
 
--   PostgreSQL versions 12, 13, 14, 15, and 16
--   EDB Postgres Advanced Server versions 12, 13, 14, 15, and 16
+-   PostgreSQL versions 12, 13, 14, 15, 16 and 17
```
Comment (zahid75) [inline product_docs/docs/eprs/7/01_introduction/03_certified_supported_versions.mdx:7]: We maintain last 5 major versions of PostgreSQL, so with the addition of v17, v12 should be dropped.

File: product_docs/docs/eprs/7/01_introduction/03_certified_supported_versions.mdx [hunk 3]
```diff
@@ -4,8 +4,8 @@ title: "Certified and supported product versions"
...
+-   EDB Postgres Advanced Server versions 12, 13, 14, 15, 16 and 17
```
Comment (zahid75) [inline product_docs/docs/eprs/7/01_introduction/03_certified_supported_versions.mdx:8]: We maintain last 5 major versions of EPAS, so with the addition of v17, v12 should be dropped.

**Review by zahid75 — CHANGES_REQUESTED** [review]
Please see my comments inline.

**Comment by gvasquezvargas** [comment]
@zahid75 updated to drop v12 support. Also, I rolled back the change to the certified PGD version back to 5.3.

**Review by mlewandowski-edb — APPROVED** [review]

**Pull request merged**